### DISCUSSION TITLE
Shadowsocks 2022

### DISCUSSION THREAD

**Discussion by AlamiShayan** [jinwyp/one_click_script#174] [Ideas]
Hi 
It would be great to add shadowsocks 2022(when it got the stable release) instead or alongside of Trojan protocol. Trojan doesn't got the maintenance that much. If shadowsocks 2022 added to this script this would be awesome and there is actually no need for anything else. 
Edit 1: shadowsocks rust is actually the best out there.seems much more active than the others. Just a suggestion.

**Comment by jinwyp** [comment]
Yes.  I will add  shadowsocks 2022 in next month.  Easily switch core for  shadowsocks-rust / Xray-core.   Waiting for xray 1.6 release

**Reply by AlamiShayan** [reply]
does the script support the new version 5 of v2ray? and the new vlite protocol? thanks for your awesome work mate will do a donation soon.

**Reply by jinwyp** [reply]
Not support V2ray 5 yet.  V2ray 5 is still at preview stage , waiting for the major release

**Reply by AlamiShayan** [reply]
sorry for bothering you, any update on shadow 2022

**Comment by jinwyp** [comment]
Added Shadowsocks Rust (Select 6)  and Shadowsocks  Xray (Select 7) . 
![readme_en](https://user-images.githubusercontent.com/656913/197366198-13f657f7-f6a4-412e-94dd-8a335c560bcb.png)
